apps/juxtaposition-ui/src/services/juxt-web/routes/console/feed.tsx

```diff
@@ -0,0 +1,167 @@
+import express from 'express';
+import { z } from 'zod';
+import { database } from '@/database';
+import { getCommunityHash } from '@/util';
+import { POST } from '@/models/post';
+import { config } from '@/config';
+import { parseReq } from '@/services/juxt-web/routes/routeUtils';
+import { WebGlobalFeedView, WebPersonalFeedView } from '@/services/juxt-web/views/web/feed';
+import { buildContext } from '@/services/juxt-web/views/context';
+import { WebPostListView } from '@/services/juxt-web/views/web/postList';
+import { CtrGlobalFeedView, CtrPersonalFeedView } from '@/services/juxt-web/views/ctr/feed';
+import { CtrPostListView } from '@/services/juxt-web/views/ctr/postList';
+import { PortalGlobalFeedView, PortalPersonalFeedView } from '@/services/juxt-web/views/portal/feed';
+import { PortalPostListView } from '@/services/juxt-web/views/portal/postList';
+
+export const feedRouter = express.Router();
+
+feedRouter.get('/', async function (req, res) {
+	const { auth, query } = parseReq(req, {
+		query: z.object({
+			pjax: z.string().optional()
+		})
+	});
+	const userContent = await database.getUserContent(auth().pid);
+	const communityMap = getCommunityHash();
+	if (!userContent) {
+		return res.redirect('/404');
+	}
+	const posts = await database.getNewsFeed(userContent, config.postLimit);
+
+	const bundle = {
+		posts,
+		open: true,
+		communityMap,
+		userContent,
+		nextLink: `/feed/more?offset=${posts.length}&pjax=true`
+	};
```

Comment:
Is there any point having "bundle" here instead of just local variables, given you pull all the members back out again during rendering?
Is it referenced somehow behind the scenes?

(e.g. communityMap seems unused here, the post listings get this out of props.ctx)

Reply:
`bundle` was so I didn't modify too much of the original route handlers. But I do agree that we can replace it by now.